claat/util/util_test.go

```diff
@@ -37,3 +37,55 @@ func TestUnique(t *testing.T) {
 		})
 	}
 }
+
+func TestNormalizedSplit(t *testing.T) {
```

Comment:
Add a test case with some Unicode-defined spaces that are not U+0020

Reply:
Done